site/documentation/content/admin-guide/hono-kafka-client-configuration.md

```diff
@@ -50,108 +58,89 @@ Kafka based APIs. The producers created by the factory are configured with insta
 configure a producer.
 
-The configuration needs to be provided in the form `HONO_KAFKA_PRODUCERCONFIG_${PROPERTY}` as an environment variable or
-as a Java system property in the form `hono.kafka.producerConfig.${property}`, where `${PROPERTY}` respectively
-`${property}` is any of the Kafka client's [producer properties](https://kafka.apache.org/documentation/#producerconfigs).
-The provided configuration is passed directly to the Kafka producer without Hono parsing or validating it.
+The configuration needs to be provided in the form `HONO_KAFKA_${CLIENTNAME}_PRODUCERCONFIG_${PROPERTY}` as an
+environment variable or as a Java system property in the form `hono.kafka.${clientName}.producerConfig.${property}`,
+where `${PROPERTY}` respectively `${property}` is any of the Kafka client's
+[producer properties](https://kafka.apache.org/documentation/#producerconfigs)
+and `${CLIENTNAME}` respectively `${clientName}` is the name of the client to be configured, as documented in the
+component's admin guide.
 
-The following properties can _not_ be set using this mechanism because the protocol adapters use fixed values instead 
-in order to implement the message delivery semantics defined by Hono's Telemetry and Event APIs.
+The following properties can _not_ be set because
+`org.eclipse.hono.client.kafka.producer.MessagingKafkaProducerConfigProperties` uses fixed values instead in order to
+implement the message delivery semantics defined by Hono's Telemetry and Event APIs.
 
 | Property Name     | Fixed Value |
 | :---------------- | :---------- |
 | `key.serializer`    | `org.apache.kafka.common.serialization.StringSerializer` |
 | `value.serializer`   | `io.vertx.kafka.client.serialization.BufferSerializer` |
 | `enable.idempotence` | `true` |
 
-{{% notice tip %}}
-The Kafka client requires at least the `bootstrap.servers` property to be set. This is the minimal configuration
-required to enable Kafka based messaging.
-{{% /notice %}}
-
-### Using TLS
-
-The factory can be configured to use TLS for authenticating the brokers in the Kafka cluster during connection
-establishment and optionally for authenticating to the broker using a client certificate.
-To use this, a Kafka Producer configuration as described in
-[Kafka documentation - section "Security"](https://kafka.apache.org/documentation/#security_configclients) needs to be
-provided. The properties must be prefixed with `HONO_KAFKA_PRODUCERCONFIG_` and `hono.kafka.producerConfig.` respectively as
-shown in [Producer Configuration Properties]({{< relref "#producer-configuration-properties" >}}).
-The complete reference of available properties and the possible values is available in 
-[Kafka documentation - section "Producer Configs"](https://kafka.apache.org/documentation/#producerconfigs).
+If the property `client.id` is provided, its value is appended by a generated unique suffix that provides additional
+information.
```

Comment:
To also cover the "if it's not provided" case, I would suggest using something like this instead:
> Kafka clients used in Hono will get a unique client identifier, containing client name and component identifier. If the property `client.id` is provided, its value will be used as prefix for the created client identifier.

Reply:
Done.